### PR TITLE
Statistics Recipe

### PR DESCRIPTION
**Pull request by LostMojo**

### CODE REVIEW

**Review by kswanson33 — COMMENTED** [review]
This looks good! Two questions: 

1. What's up with the two `.DS_Store` binary files in this PR?
2. How does the `PROFILE` run look? Since the stats books are longer, it would be good to check!

**Comment by LostMojo** [comment]
@kswanson33 1. Those shouldn't be there-- I was moving some stuff around in finder, it's probably to do with that :T
2. Looks like I could maybe tighten the selectors for eoc sections a little

File: books/statistics/bake [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+      uuid_key: '.bring-together-homework',
+      section_selector: 'section.bring-together-homework',
+      append_to: chapter
+    ) do |section|
+      RemoveSectionTitle.v1(section: section)
+    end
+
+    BakeChapterReferences.v1(chapter: chapter, metadata_source: book.metadata)
+
+    exercise_selectors = '.practice, .bring-together-exercises, .free-response, .bring-together-homework'
```
Comment (kswanson33) [inline books/statistics/bake:102]: didn't see these yesterday -- could you add tag names (I assume these are `section`s) to speed up the selectors? :)

**Review by kswanson33 — CHANGES_REQUESTED** [review]
Just one comment, otherwise looks good!

File: books/statistics/bake (hunk 1 above)
Comment (LostMojo) [reply books/statistics/bake]: Done!

**Review by kswanson33 — APPROVED** [review]

**Pull request merged**